script/update_jito_validators.rb

```diff
@@ -9,7 +9,7 @@
 if response.is_a?(Net::HTTPSuccess)
   json_response = JSON.parse(response.body)
   jito_vote_accounts = json_response["validators"].map do |jito_validator|
-    jito_validator["vote_account"] if jito_validator["running_jito"]
+    [jito_validator["vote_account"], jito_validator["mev_commission_bps"]] if jito_validator["running_jito"]
   end
```

Comment:
Let's use a Hash instead of an Array. That would remove some further logic below.

 ```
  jito_vote_accounts = json_response["validators"].map do |validator|
{ validator["vote_account"] => validator["mev_commission_bps"] } if validator["running_jito"]
end

```